docs/app/app.component.html

```diff
@@ -1,8 +1,19 @@
-<uxd-navigation-bar></uxd-navigation-bar>
+<uxd-navigation-bar *ngIf="isNavigationVisible"></uxd-navigation-bar>
 
-<div class="app-page-content">    
+<ux-page-header
```

Comment:
As with the Micro Focus site, it would helpful to add a comment indicating that this is only for the showcase.